keadm/cmd/keadm/app/cmd/common/constant.go

```diff
@@ -50,6 +50,9 @@ const (
 
 	// FlagNamePrintFinalValues ...
 	FlagNamePrintFinalValues = "print-final-values"
+
+	// FlagNamePostRun ...
+	FlagNamePostRun = "post-run"
```

Comment:
why is this assigned to `Cloud init` flag constant? according to https://github.com/kubeedge/kubeedge/issues/5940, it is supposed to be under `Edge join flag names`? Or if those are common flags, they can be under `Common flag names`?

Reply:
Thank you for your suggestion. I have modified it.